app/presenters/vendor_api/base.rb

```diff
@@ -53,7 +53,14 @@ def version_available_in_environment?(version)
     end
 
     def cache_key(model, api_version, suffixes = {})
-      CacheKey.generate("#{api_version}_#{model.cache_key_with_version}#{suffixes.hash}")
+      suffixes_name = suffixes.values.compact_blank.join('-')
```

Comment:
For another PR but Is `app/lib/cache_key.rb` needed anymore since we are generating the key this way now?

Or shall we rename the method `generate`?

Reply:
Yeah other places use this method 🤦‍♂️